internal/server/storage/drivers/driver_zfs.go

```diff
@@ -261,7 +272,12 @@ func (d *zfs) Create() error {
 		}
 
 		// Create the zpool.
-		_, err = subprocess.RunCommand("zpool", "create", "-m", "none", "-O", "compression=on", d.config["zfs.pool_name"], loopPath)
+		createArgs := []string{"create", "-m", "none", "-O", "compression=on", d.config["zfs.pool_name"]}
+		if poolType != "" {
+			createArgs = append(createArgs, poolType)
+		}
+		createArgs = append(createArgs, loopPath)
+		_, err = subprocess.RunCommand("zpool", createArgs...)
```

Comment:
This change doesn't make sense, we're dealing with the loop path case which is when the user doesn't provide  devices, so we're guaranteed to only deal with a single device which is `loopPath`, no need to change anything here.